jen/group_test.go

```diff
@@ -0,0 +1,36 @@
+package jen_test
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	. "github.com/dave/jennifer/jen"
+)
+
+func TestGroup_Render(t *testing.T) {
+	file := NewFile("main")
+	file.ImportAlias("fmt", "fmtalias")
+
+	g := &Group{
```

Comment:
To get a `Group` without having to use private fields, you could do something like:

```go
var group *Group
b := BlockFunc(g *Group) {
    group = g
}
group.RenderWithFile(...)
```

... I think that would work?

Reply:
Fixed